### PR TITLE
Integrates Metrics widgets into Static Site

### PR DESCRIPTION
**Pull request by btylerburton**
## Changes proposed in this pull request:
- Adds new catalog data and metrics visualizations 
- Upgrades 11ty to 2.x.x

## security considerations
None

### CODE REVIEW

**Review by robert-bryson — DISMISSED** [review]

**Comment by FuhuXia** [comment]
Search gives no results.
https://catalog.data.gov/dataset?search=test

File: _data/site.yaml [hunk 1]
```diff
@@ -80,24 +88,24 @@ catalog_filters:
...
-  - title: Case Studies & Examples
-    url: https://resources.data.gov/categories/case-studies-examples/
   - title: Data Inclubator
     url: https://resources.data.gov/categories/data-incubator/
+  - title: Guidance
+    url: https://resources.data.gov/categories/guidance/
+  - title: Case Studies & Examples
+    url: https://resources.data.gov/categories/case-studies-examples/
   - title: Skills Development
     url: https://resources.data.gov/categories/skills-development/
```
Comment (nickumia-reisys) [inline _data/site.yaml:100]: Reordering for better flow of readability?  Order of introduction of material?  Or some other reason?

File: _data/site.yaml [hunk 2]
```diff
@@ -80,24 +88,24 @@ catalog_filters:
...
   - title: 2021 Action Plan
     url: https://strategy.data.gov/2021/action-plan/
-  - title: 2021 Action Progress
-    url: https://strategy.data.gov/2021/progress/
   - title: 2020 Action Plan
     url: https://strategy.data.gov/2020/action-plan/
+  - title: 2021 Action Progress
+    url: https://strategy.data.gov/2021/progress/
   - title: 2020 Action Progress
     url: https://strategy.data.gov/2020/progress/
```
Comment (nickumia-reisys) [inline _data/site.yaml:110]: I think it makes more sense to have it grouped by year (chronologically) and not progress vs. plan...

File: _includes/metrics_widgets/bubble-chart.html [hunk 3]
```diff
@@ -0,0 +1,4 @@
+<div class="metrics__card {{class}}">
+    <canvas id="datagov-bubble-chart" class="datagov-charts" data-metric="{{metric}}"></canvas>
+</div>
+
```
Comment (nickumia-reisys) [inline _includes/metrics_widgets/bubble-chart.html:4]: Leaving this for future use of bubble charts?

**Comment by btylerburton** [comment]
Good catch @FuhuXia, it's fixed...

**Review by nickumia-reisys — COMMENTED** [review]
The catalog fetching code could probably be optimized a bit, but let's save that for a smaller PR focused on function performance/readability/effectiveness.

Otherwise, it's 🚢 📦

File: .eleventy.js [hunk 4]
```diff
@@ -86,21 +97,36 @@ module.exports = function (config) {
...
+        // Use a local key/certificate to opt-in to local HTTP/2 with https
+        https: {
+            // key: "./localhost.key",
+            // cert: "./localhost.cert",
         },
-        ui: false,
-        ghostMode: false,
+
+        // Change the default file encoding for reading/serving files
+        encoding: 'utf-8',
```
[the remark this answers is not in the document]
Comment (btylerburton) [reply .eleventy.js:129]: This is part of the upgrade to 11ty 2.x. I believe it was a warning output from this module: https://github.com/11ty/eleventy-upgrade-help

File: _data/site.yaml (hunk 1 above)
Comment (btylerburton) [reply _data/site.yaml:100]: I figured out how to order the items as expected with flexbox.

File: _data/site.yaml (hunk 2 above)
Comment (btylerburton) [reply _data/site.yaml:110]: Just following the comps that Carolina made. Let's get this in as a first pass then record thoughts like this on the QA ticket: https://github.com/GSA/data.gov/issues/4213

File: _includes/metrics_widgets/bubble-chart.html (hunk 3 above)
Comment (btylerburton) [reply _includes/metrics_widgets/bubble-chart.html:4]: I can take this out.

**Comment by btylerburton** [comment]
@nickumia-reisys fixed that layout bug in firefox with the latest commit...

**Review by nickumia-reisys — APPROVED** [review]
🤖

**Pull request merged**